### ISSUE TITLE
How to specify labels path for training?

### ISSUE THREAD

**Issue by ydixon** [andy-yun/pytorch-0.4-yolov3#10]
I'm trying to train the model from the COCO dataset. I've realized I can set the train/valid targets by modifying the parameters in  "coco.data". However, I don't see a place where I can set the labels target.
My labels are stored in txt where each txt file contains the label for a single image as follow:

Image: /data/coco/images/train2014/COCO_train2014_000004.jpg
label: /data/coco/labels/train2014/COCO_train2014_000004.txt

How should I proceed?

**Comment by andy-yun** [comment]
please refer to https://github.com/marvis/pytorch-yolo2 for detailed procedure.
The same procedures can be applied.

**Comment by lianuo** [comment]
@ydixon have you solve this problem? I do not find the solotion in marvis...

**Comment by andy-yun** [comment]
In marvis repo,

You can find the section as following:
Generate Labels for VOC

wget http://pjreddie.com/media/files/voc_label.py
python voc_label.py
cat 2007_train.txt 2007_val.txt 2012_*.txt > voc_train.txt

**Comment by lianuo** [comment]
@andy-yun Thank  you so much for fast reply!

**Comment by berryweinst** [comment]
Hi, is there an actual script someone wrote for labeling the 2017 training data (voc_label.py style)? I'm a bit struggling with making it work.
